data.json

```
@@ -995,4 +995,13 @@
     "imageUrl": "https://user-images.githubusercontent.com/10162710/120894576-6ff8f200-c619-11eb-9ee5-5078912d5550.png",
     "issueNumber": 552
   }
+  
+  {
+    "name": "Leticia Rosário",
+    "githubUsername": "leticiarosario",
+    "imageUrl": "(https://user-images.githubusercontent.com/60137996/121890703-c9d87680-cd01-11eb-938c-2d4e4416126a.png)",
+    "issueNumber": 554
+}
```

```suggestion
 }
```

---

```suggestion
  }
```
@starlightknown there is one space missing but great spot